.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - 4
```

Comment:
fair enough. I don't expect to explicitly drop it, it just adds a bunch of build time.

would be fine to add it back